doc/cephfs/disaster-recovery-experts.rst

```diff
@@ -316,3 +316,44 @@ operational.
     will contain holes in place of the missing data.
 
 .. _Symbolic link recovery: https://tracker.ceph.com/issues/46166
+
+
+Restoring file system after recovering monitor stores using OSDs
+----------------------------------------------------------------
+
+During rare occasions, all the monitor stores of a cluster may get corrupted.
+To recover the cluster in such a scenario, you need to rebuild the monitor
+stores using the OSDs, and get back the pools intact (active+clean state).
+However, the rebuilt monitor stores don't restore the FS/MDS maps. Additonal
+steps are required to bring back the file system. Currently, only the steps to
+recover a **single active MDS** file system have been identified, and tested.
+These are described below.
+
```

Comment:
I think it would be helpful to outline the general idea of the recovery: we're creating the FSMap with some basic defaults (we cannot necessarily recovery every setting applied) and allowing the MDS to recover from what is in the journal/metadata of the pools.